src/css/styles.css

```diff
@@ -102,7 +91,8 @@ body {
     box-shadow: 0px 2px 5px rgba(100, 100, 100, 0.7);
     flex: 0 0 auto;
     margin-bottom: 0;
-    position: fixed;
+    position: sticky;
```

Comment:
You need the `-webkit-` prefix for WebKit, sadly, so:

```css
position: -webkit-sticky;
position: sticky;
```

Reply:
Will do